packages/bpk-component-card/src/BpkDividedCard.tsx

```diff
@@ -42,7 +42,6 @@ export type Props = {
 };
 
 const BpkDividedCard = ({
-  className = null,
```

Comment:
Although className has been removed here, it will still be passed into BpkCard within the `{...rest}` props. This is unavoidable and is required to persist to support existing `className` usage.